R/api.R

```diff
@@ -59,6 +59,31 @@
 #' key_get("R-keyring-test-service", "donaldduck")
 #' if (has_keyring_support()) key_list(service = "R-keyring-test-service")
 #' key_delete("R-keyring-test-service", "donaldduck")
+#'
+#' ## This is interactive using backend_file
+#' library(keyring)
+#' 
+#' ## Set variables to be used in keyring
+#' kr_name <- "my_keyring"
+#' kr_service <- "my_database"
+#' kr_username <- "my_username"
+#' 
+#' ## Create a keyring and add an entry using the variables above
+#' kb <- keyring::backend_file$new()
+#' ## Prompt for the keyring password, used to unlock keyring
+#' kb$keyring_create(kr_name)
+#' ## Prompt for the secret/password to be stored in the keyring
+#' kb$set(kr_service, username=kr_username, keyring=kr_name)
+#' # Encrypt the keyring
+#' kb$keyring_lock(kr_name)
+#' 
+#' ## The encrypted keyring file is now stored at ~/.config/r-keyring/ and can be
```

Comment:
Yeah, either we need to use a temp file, or move this example out of `@examples`, because this is not allowed on CRAN.

Also, the location is different on other platforms, it is in `.config` only on Linux.

Reply:
Good point, I could add that it's only on linux.  Since this is in `\dontrun{}`, I think the example will not be run and the file will not be created when building on CRAN.